### PR TITLE
Additional information about working with PreHooks

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.6.0

Resolves https://issues.redhat.com/browse/MTV-448 by adding a note about usage with PreHooks.

Preview: https://file.emea.redhat.com/rhoch/prehook_addition/html-single/#adding-hooks-migration-plan-using-api

### CODE REVIEW

**Comment by RichardHoch** [comment]
@ahadas Please review.

**Review by ahadas — APPROVED** [review]

File: documentation/modules/about-hook-crs-for-migration-plans-api.adoc [hunk 1]
```diff
@@ -22,6 +22,11 @@ You specify hooks per VM and you can run each as a _PreHook_ or a _PostHook_. In
 
 When you add a hook, you must specify the namespace where the hook CR is located, the name of the hook, and specify whether the hook is a PreHook or PostHook.
 
+[IMPORTANT]
+====
+In order for a PreHook to run on a VM, the VM must be started and available by SSH.
```
Comment (anarnold97) [inline documentation/modules/about-hook-crs-for-migration-plans-api.adoc:27]:
```suggestion
In order for a PreHook to run on a VM, the VM must be started and available using Secure Shell Protocol (SSH).
```
i have a bit of issue with the preposition `by`
Do not care if you go with just `SSH`

Comment (anarnold97) [inline documentation/modules/about-hook-crs-for-migration-plans-api.adoc:27]:
```suggestion
In order for a PreHook to run on a VM, the VM must be started and available via SSH.
```

File: documentation/modules/adding-hook-crs-to-migration-plans-api.adoc [hunk 2]
```diff
@@ -176,4 +174,9 @@ spec:
             name: playbook
           step: PreHook <1>
 ----
-<1> Options are `PreHook`, to run the hook before the migration, and `PostHook`, to run the hook after the migration.
\ No newline at end of file
+<1> Options are `PreHook`, to run the hook before the migration, and `PostHook`, to run the hook after the migration.
+
+[IMPORTANT]
+====
+In order for a PreHook to run on a VM, the VM must be started and available by SSH.
```
Comment (anarnold97) [inline documentation/modules/adding-hook-crs-to-migration-plans-api.adoc:181]:
```suggestion
In order for a PreHook to run on a VM, the VM must be started and available via SSH.
```

Comment (anarnold97) [reply documentation/modules/adding-hook-crs-to-migration-plans-api.adoc:181]: See above

**Comment by anarnold97** [comment]
Other than a pedantic preposition suggestion. LGTM

**Comment by RichardHoch** [comment]
"via" it is.

**Pull request merged**